stock_picking_package_preparation_value/models/stock_ppp_value.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Andrea Cometa - Apulia Software
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/gpl.html).
+
+from openerp import models, fields, api
+from openerp.addons.decimal_precision import decimal_precision as dp
+
+
+class StockPickingPackagePreparation(models.Model):
+    _inherit = 'stock.picking.package.preparation'
+
+    @api.one
+    def _compute_amount(self):
+        self.amount_untaxed = 0.0
+        self.amount_tax = 0.0
+        self.amount_total = 0.0
+        for picking in self.picking_ids:
+            self.amount_untaxed += picking.amount_untaxed
+            self.amount_tax += picking.amount_tax
+            self.amount_total += picking.amount_total
+
+    amount_untaxed = fields.Float(
+        compute='_compute_amount', digits_compute=dp.get_precision('Account'),
+        string='Untaxed Amount')
```

Comment:
@scigghia I think it can be useful/safe to have 2 set of fields (one for sale one for purchase), invisible depending on the context.
Additional you could add a link to the resource it is based from ("SO0001" or "PO0002").
You could reuse the fields in report with clear title (Sales amount or Purchase Amount).

Nevertheless, I am always reluctant to show sales or purchase value to logistic people. Most of the time they should not be aware/allowed of this information. For this reason I would suggest that you add a ACL group "See amount in SM" for example) so that you can display the information to only specific people.